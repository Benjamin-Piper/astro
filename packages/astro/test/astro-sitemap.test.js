import { createBuilder } from './helpers.js';

let builder;

describe('Sitemap Generation', () => {
  beforeAll(async () => {
    builder = await createBuilder('./fixtures/astro-rss');
    await builder.build();
  });

  test('Generates Sitemap correctly', async () => {
    let sitemap = await builder.readFile('/sitemap.xml');
    expect(sitemap).toMatchSnapshot();
  });
});
