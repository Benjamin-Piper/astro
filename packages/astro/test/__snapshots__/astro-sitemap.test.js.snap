// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sitemap Generation Generates Sitemap correctly 1`] = `
"<?xml version=\\"1.0\\" encoding=\\"UTF-8\\"?><urlset xmlns=\\"http://www.sitemaps.org/schemas/sitemap/0.9\\"><url><loc>https://mysite.dev/episode/fazers/</loc></url><url><loc>https://mysite.dev/episode/rap-snitch-knishes/</loc></url><url><loc>https://mysite.dev/episode/rhymes-like-dimes/</loc></url><url><loc>https://mysite.dev/episodes/</loc></url></urlset>
"
`;
